blog/scss/layout/_footer.scss

```diff
@@ -0,0 +1,226 @@
+.weprodev-footer {
+  padding-top: nth($spaces, 24);
+  color: $white;
+  line-height: 1.7;
+  font-weight: $fontWeightRegular;
+  background-image: linear-gradient($obsidianGlow, $obsidianGlow);
+  background-size: cover;
+
+  .containerA {
+    width: 70%;
+    margin: 0 auto;
+    display: grid;
+    justify-content: space-between;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-gap: 0 nth($spaces, 4);
+    justify-content: space-between;
+
+    .WeProDev,
+    .ContactInfo,
+    .FollowUs {
```

Comment:
when you want to follow the code convention?! do you have any plan in your mind?!